aequilibrae/paths/cython/graph_building.pyx

```diff
@@ -442,15 +469,19 @@ def create_compressed_link_network_mapping(graph):
                 j += 1
 
             b_node = x
-            node_mapping[a_node] = graph.compact_graph["a_node"].iat[compressed_id]
-            node_mapping[b_node] = graph.compact_graph["b_node"].iat[compressed_id]
-
-            i += len(values)
+            node_mapping[a_node] = compact_a_nodes[compressed_id]
+            node_mapping[b_node] = compact_b_nodes[compressed_id]
+        i += count
 
-        idx[-1] = i
+    idx[-1] = i
 
-        graph.compressed_link_network_mapping_idx = np.array(idx)
-        graph.compressed_link_network_mapping_data = np.array(data)
-        graph.network_compressed_node_mapping = np.array(node_mapping)
+    graph.compressed_link_network_mapping_idx = np.array(idx)
+    graph.compressed_link_network_mapping_data = np.array(data)
+    graph.network_compressed_node_mapping = np.array(node_mapping)
 
-        return idx, data, node_mapping
+    print("end compressed link network mapping", time.time() - start_tiem)
```

Comment:
@Jake-Moss , let's remove this printing.  MMaybe put as a logging.debug?

Reply:
Whoops I have a commit for that just mustn't have pushed it